coffee-dto/coffee-dto-stub-gen/pom.xml

```diff
@@ -76,44 +82,112 @@
 					</execution>
 				</executions>
 			</plugin>
-			<plugin>
-				<groupId>com.github.os72</groupId>
-				<artifactId>protoc-jar-maven-plugin</artifactId>
-				<version>3.11.4</version>
-				<configuration>
-					<protocArtifact>com.google.protobuf:protoc:${version.protoc.version}:exe:${os.detected.classifier}</protocArtifact>
-					<inputDirectories>
-						<!-- only own coffee proto files generate -->
-						<include>${project.build.directory}/proto-external</include>
-					</inputDirectories>
-					<includeMavenTypes>transitive</includeMavenTypes>
-					<includeDirectories>
-						<directory>${project.build.directory}/proto-external</directory>
-					</includeDirectories>
-					<outputTargets>
-						<outputTarget>
-							<type>java</type>
-						</outputTarget>
-						<outputTarget>
-							<type>grpc-java</type>
-							<pluginArtifact>io.grpc:protoc-gen-grpc-java:${version.io.grpc.version}:exe:${os.detected.classifier}</pluginArtifact>
-						</outputTarget>
-						<outputTarget>
-							<type>grpc-coffee</type>
-							<pluginArtifact>hu.icellmobilsoft.coffee:coffee-grpc-protoc:${project.version}</pluginArtifact>
-						</outputTarget>
-					</outputTargets>
-				</configuration>
-				<executions>
-					<execution>
-						<phase>generate-sources</phase>
-						<goals>
-							<goal>run</goal>
-						</goals>
-					</execution>
-				</executions>
-			</plugin>
 		</plugins>
 	</build>
+	
+    <profiles>
+        <profile>
+            <id>arm64-protoc-java-gen-with-preistalled-executable</id>
+            <activation>
+                <property>
+                    <!-- add this property to settings.xml -->
+                    <name>cpu.arch</name>
+                    <value>aarch_64</value>
```

Comment:
https://maven.apache.org/guides/introduction/introduction-to-profiles.html#OS
`<arch />` kapcsoloval nem megy?

Reply:
(y) kiprobalom